ncdump/CMakeLists.txt

```diff
@@ -263,6 +263,7 @@ endif()
     add_sh_test(ncdump tst_ncgen4)
     add_sh_test(ncdump tst_netcdf4_4)
     add_sh_test(ncdump tst_nccopy4)
+    add_sh_test(ncdump tst_calendars_nc4)
```

Comment:
Adding this same test to the autotools-based build as well.

Reply:
Sorry I missed this, and thanks for fixing it!